src/app/containers/CpsTable/CpsTableCell/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Yeah I don't find the distinction useful - I think the whole "containers and components" pattern was popularised by Dan Abramov in a 6-year-old medium post that went viral: https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0

He has since updated it to say he doesn't consider it a useful pattern - I tend to agree.  Particularly in Simorgh, it seems somewhat arbitrary where we put components.  Reason I put this one in containers is really just because that's where all the other block renderers are

Reply:
Thanks @ryanmccombe, just wanted to sound this out, I'd be in favour flattening the structure to just `components` we should bring this up with the wider team. Just thinking about this stuff before we do live pages where we will make a lot more components.

I'd stick with this in containers like you've done for now 👍

Reply:
I agree! https://github.com/bbc/simorgh/issues/4478